spec/chromium-spec.js

```diff
@@ -1312,3 +1313,59 @@ describe('chromium feature', () => {
     })
   })
 })
+
+describe('font fallback', () => {
+  async function getRenderedFonts (html) {
+    const w = new BrowserWindow({ show: false })
+    try {
+      const loaded = emittedOnce(w.webContents, 'did-finish-load')
+      w.loadURL(`data:text/html,${html}`)
+      await loaded
+      w.webContents.debugger.attach()
+      const sendCommand = (...args) => new Promise((resolve, reject) => {
+        w.webContents.debugger.sendCommand(...args, (e, r) => {
+          if (e) { reject(e) } else { resolve(r) }
+        })
+      })
+      const { nodeId } = (await sendCommand('DOM.getDocument')).root.children[0]
+      await sendCommand('CSS.enable')
+      const { fonts } = await sendCommand('CSS.getPlatformFontsForNode', { nodeId })
+      return fonts
+    } finally {
+      w.close()
+    }
+  }
+
+  it('should use Helvetica for sans-serif on Mac, and Arial on Windows and Linux', async () => {
+    const html = `<body style="font-family: sans-serif">test</body>`
+    const fonts = await getRenderedFonts(html)
+    expect(fonts).to.be.an('array')
+    expect(fonts).to.have.length(1)
+    expect(fonts[0].familyName).to.equal({
+      'win32': 'Arial',
+      'darwin': 'Helvetica',
+      'linux': 'DejaVu Sans' // I think this depends on the distro? We don't specify a default.
```

Comment:
Is that really a default font on linux?  I thought you had to install those powerline fonts manually 🤔

Reply:
> DejaVu fonts can be obtained from the DejaVu project on GitHub. Some operating systems (OpenBSD, Solaris, Haiku, AmigaOS 4, GNU/Linux distributions such as Ubuntu, Debian, Fedora, and RHEL) include DejaVu fonts in their default installation,[3][4][5][6][7] sometimes even using them as their system fonts

https://en.wikipedia.org/wiki/DejaVu_fonts